src/BenchmarkDotNet/Columns/DefaultColumnProvider.cs

```diff
@@ -63,13 +63,23 @@ public IEnumerable<IColumn> GetColumns(Summary summary)
                     bool hide = stdDevColumnValues.All(value => value == "0.00" || value == "0.01");
                     if (!hide)
                         yield return BaselineRatioColumn.RatioStdDev;
+
+                    if (HasDisassemblyDiagnoser(summary))
+                    {
+                        yield return new BaselineAllocationRatioColumn();
+                    }
                 }
             }
 
             private static bool NeedToShow(Summary summary, Func<Statistics, bool> check)
             {
                 return summary.Reports != null && summary.Reports.Any(r => r.ResultStatistics != null && check(r.ResultStatistics));
             }
+
+            private static bool HasDisassemblyDiagnoser(Summary summary)
```

Comment:
Disassembly -> Memory

```suggestion
                    if (HasMemoryDiagnoser(summary))
                    {
                        yield return new BaselineAllocationRatioColumn();
                    }
                }
            }

            private static bool NeedToShow(Summary summary, Func<Statistics, bool> check)
            {
                return summary.Reports != null && summary.Reports.Any(r => r.ResultStatistics != null && check(r.ResultStatistics));
            }

            private static bool HasMemoryDiagnoser(Summary summary)
```